core-common/src/main/java/eu/arrowhead/common/CoreCommonConstants.java

```diff
@@ -69,6 +70,7 @@
 	public static final String OP_SERVICE_REGISTRY_QUERY_SERVICES_BY_SYSTEM_ID_URI = "/query/provider/{" + COMMON_FIELD_NAME_ID + "}";
 	public static final String OP_SERVICEREGISTRY_MULTI_QUERY_URI = CommonConstants.OP_SERVICEREGISTRY_QUERY_URI + "/multi";	
 	public static final String OP_SERVICE_REGISTRY_QUERY_SERVICES_BY_SERVICE_DEFINITION_LIST_URI = "/query/definition";
+	public static final String OP_SERVICEREGISTRY_PULL_CONFIG_URI = "/pull-confing";
```

Comment:
you should fix this, before any other Hungarian people see it. :)

Reply:
jesus... :)